marlowe-playground-server/contracts/ContractForDifferences.hs

```diff
@@ -0,0 +1,103 @@
+{-# LANGUAGE NumericUnderscores #-}
```

Comment:
Why are these contracts on the playground-server?

Reply:
They are converted to Marlowe by the PSGenerator